src/main/java/org/symphonyoss/symphony/messageml/elements/PersonSelector.java

```diff
@@ -14,28 +21,109 @@
 public class PersonSelector extends FormElement {
   public static final String MESSAGEML_TAG = "person-selector";
 
+  private static final String PLACEHOLDER_ATTR = "placeholder";
+
   private static final String PRESENTATIONML_TAG = "div";
-  private final static String MARKDOWN = "Person Selector";
+  private static final String PRESENTATIONML_PLACEHOLDER_ATTR = "data-placeholder";
+
+  private final static String MARKDOWN = "Person Selector:";
   private static final String CLASS_ATTR = "class";
+  private static final String PRESENTATIONML_NAME_ATTR = "data-name";
 
   public PersonSelector(Element parent) {
     super(parent, MESSAGEML_TAG);
   }
 
+  @Override
+  public void buildAll(MessageMLParser context, org.w3c.dom.Element element) throws InvalidInputException, ProcessingException {
+    switch (getFormat()) {
+      case MESSAGEML:
+        super.buildAll(context, element);
+        break;
+      case PRESENTATIONML:
+        buildElementFromPresentationML(context, element);
```

Comment:
Same here, you are building from the div, not from the presentation. The format is presentation, but you're building from the "div" element.